source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
+                     -H "Authorization: Bearer <access_token>"
+
+                  .. output::
+                     :language: json
+                     :emphasize-lines: 4-5
+
+                     [
+                     {
+                        "cloud_provider_region": "aws-us-east-1",
+                        "service_name": "com.amazonaws.vpce.us-east-1.vpce-svc-0718bec2c0bdeab5c",
+                        "endpoint_type": "sync"
+                     },
+                     ...
+                     ]
+            #. Get the ``service_name`` field from the desired entry in the
+               previous step.
+            #. Create a VPC Endpoint using the AWS CLI `create-vpc-endpoint 
+               <https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc-endpoint.html>`__
+               command. You need an App Services endpoint service name, VPC ID,
+               subnet IDs, and AWS deployment region. The command should look
+               something like this:
+                  
             .. code-block:: sh
 
-               aws ec2 modify-vpc-endpoint \
-                 --region <your-aws-deployment-region> \
-                 --vpc-endpoint-id <your-vpc-endpoint-id> \
-                 --private-dns-enabled
+              aws ec2 create-vpc-endpoint \
+                --vpc-endpoint-type Interface
+                --service-name <App Services Endpoint Service Name> \
+                --vpc-id <your-vpc-id> \
+                --region <your-aws-deployment-region> \
+                --subnet-ids <your-subnet-ids>
+
+         .. step:: Add the Private Endpoint to Your App
+
+            Now that you've created the interface endpoint in your VPC, you
+            need to add it to your App. 
+
+            a. Find the Sync **VPC Endpoint ID**, either in the
+               output of the CLI commands you ran in the previous step or on the
+               endpoint details screen of the AWS VPC dashboard. This value
+               starts with ``vpce-``.
+            #. Create a Sync Private Endpoint by calling
+               :admin-api-endpoint:`Create a VPC Private Endpoint <operation/adminCreatePrivateEndpoint>`
+               with its Endpoint ID, cloud deployment region, and endpoint type.
+               For example, using ``aws-us-east-1``:
+
+               .. code-block:: sh
+
+                  curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/private_endpoints \
+                  -X POST \
+                  -H "Authorization: Bearer <accessToken>" \
+                  -H "Content-Type: application/json" \
+                  -d '{
+                     "cloud_provider_region": "<your-cloud-deployment-region>",
+                     "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                     "endpoint_type": "sync",
+                     "comment": "<comment-about-endpoint>"
```

Comment:
Same Q as above about adding a more descriptive comment here since we're divorced from the Admin API definition.